src/server/administer/index.md

```diff
@@ -8,7 +8,25 @@ Administration guide will help you to configure and maintain your <CommunityPlat
 
 See the list of all [environment variables](./environment.md) that you can configure.
 
-## Opt out of sending statistics
+## Telemetry Service
+
+<MainPlatformName /> telemetry service helps us gather some information about the usage of the platform, namely on-premise deployment. This information is very valuable to us to help us improve <MainPlatformName />.
```

Comment:
```suggestion
The telemetry service helps us gather some information about the usage of the platform, namely on-premise deployment. This information is very valuable to us to help us improve <MainPlatformName />.
```

Unfortunately, there is a thing in vuepress that if the paragraph starts with a component, it won't be formatted properly. So you need to either write "Mergin Maps" instead of using the component or rephrase the sentence.

Reply:
Also add: 
* Which information do you get from this licence monitoring (users seat, URL, uuid) ?
* How much time do you keep it? (unlimited)